src/create-cluster.spec.ts

```diff
@@ -57,9 +57,9 @@ test('expect error is cli returns non zero exit code', async () => {
   try {
     (extensionApi.process.exec as Mock).mockReturnValue({ exitCode: -1, error: 'error' });
     await createCluster({}, undefined, '', telemetryLoggerMock);
-  } catch (err) {
+  } catch (err: unknown) {
     expect(err).to.be.a('Error');
-    expect(err.message).equal('Failed to create minikube cluster. error');
+    expect((err as Error).message).equal('Failed to create minikube cluster. error');
```

Comment:
issue: we should not do any casting if it's not possible. So first, check that err is instanceof Error (or has message field)

Reply:
Addressed by https://github.com/podman-desktop/extension-minikube/pull/225